tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/core/SentenceTagger.scala

```diff
@@ -190,7 +192,7 @@ object SentenceTagger {
       for {
         sentence <- sentenceSource.sentenceIterator
       } yield Future {
-        tagger.tag(sentence)
+        tagger.tag(sentence, Set())
```

Comment:
This utility function is for unconstrained tagging (I added a comment). It could support constraints, but it's not a likely use case in the near future.